.env

```diff
@@ -12,7 +12,7 @@ OFF_GID=1000
 # Robotoff
 TAG=latest
 ROBOTOFF_INSTANCE=localhost
-ROBOTOFF_DOMAIN=openfoodfacts.localhost
+ROBOTOFF_DOMAIN=openfoodfacts.net
```

Comment:
@kulizhsy let's keep localhost here, it will be the most used case

Reply:
```suggestion
ROBOTOFF_DOMAIN=openfoodfacts.localhost
```

Reply:
Why will it be the most used case?

If we keep it as `openfoodfacts.localhost`, we would not be able to run Robotoff locally since this var is used both for a loopback to robotoff APIs and to fetch data from PO (which is not gonna be running locally most likely).

I'm fine reverting it to `localhost` in this PR, but #485 is just going to flip it back to `net` since we can't run integration tests otherwise.

The actual solution here is to decouple the usage of `ROBOTOFF_DOMAIN` into 1) Loopback to localhost for things that need to be resolved locally, and 2) Remote domain local Robotoff can access for things that are not available locally.